bulbs/super_features/serializers.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import serializers
+
+from bulbs.content.models import Content
+from bulbs.content.serializers import ContentSerializer
+from bulbs.super_features.models import BaseSuperFeature, ContentRelation
+from bulbs.super_features.utils import get_data_serializer
+
+
+class ContentRelationSerializer(serializers.ModelSerializer):
+    parent_id = serializers.PrimaryKeyRelatedField(
+        source='parent', queryset=Content.objects.all()
+    )
+    child_id = serializers.PrimaryKeyRelatedField(
+        source='child', queryset=Content.objects.all()
+    )
+
+    class Meta:
+        model = ContentRelation
+        fields = ('id', 'parent_id', 'child_id', 'ordering')
+
+
+class BaseSuperFeatureDataField(serializers.Field):
+
+    def to_internal_value(self, data):
+        serializer = get_data_serializer(
```

Comment:
I'd call this variable `serializer_class` to be consistent with `to_representation` below